aspnetcore/tutorials/razor-pages/new-field.md

```diff
@@ -45,13 +45,13 @@ Update the following pages:
 * Update [Create.cshtml](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Pages/Movies/Create.cshtml) with a `Rating` field.
 * Add the `Rating` field to the Edit Page.
 
-The app won't work until the DB is updated to include the new field. If run now, the app throws a `SqlException`:
+The app won't work until the DB is updated to include the new field. Running the app now would throw a `SqlException`:
```

Comment:
Avoid future tense.

Reply:
Got it - look like I need more time with a style guide :)